content/en/apps/guides/performance/replication.md

```diff
@@ -37,10 +39,108 @@ For example:
 }
 ```
 
+##### Code sample 3.10 and later:
+```json
+{
+  "replication_depth": [
+    { "role": "district_manager", "depth": 1, "report_depth": 1 },
+    { "role": "national_manager", "depth": 2 }
+  ]
+}
+```
```

Comment:
I propose only including the code sample from `master` and linking to the app settings reference that explains the schema in more detail.

Reply:
Done.